armory/scenarios/poisoning_gtsrb_scenario.py

```diff
@@ -100,32 +104,42 @@ def _evaluate(self, config: dict) -> dict:
             f"Fitting model {model_config['module']}.{model_config['name']} "
             f"for defense {defense_config['name']}..."
```

Comment:
This will error if defense is null, which would likely be the case of filtering defense was not used.